bin/minifollowups/pycbc_page_coincinfo

```diff
@@ -145,6 +148,7 @@ for fname in args.single_trigger_files:
 
 bank = h5py.File(args.bank_file, 'r')
 statmapfile = d
+data = []
```

Comment:
comment on what 'data' is going to be holding?

Reply:
Added requested comment.